x-pack/plugins/transform/public/app/common/transform_list.ts

```diff
@@ -24,6 +24,10 @@ export interface TransformListRow {
   alerting_rules?: TransformHealthAlertRule[];
 }
 
+export type TransformWithoutConfig =
+  | TransformListRow
+  | { id: string; config: undefined; stats: undefined };
```

Comment:
should it be 
```suggestion
export type TransformWithoutConfig = Omit<TransformListRow, 'config' | 'stats'>
```
instead?

Reply:
I ended up removing the Delete modal and no longer need to do this :)